terraform/deployments/variables/integration/common.tfvars

```diff
@@ -30,7 +30,7 @@ external_dns_subdomain    = "eks"
 www_dns_validation_rdata  = "8xpwlbcbmg9qjx9d2v.fastly-validations.com"
 
 frontend_memcached_node_type   = "cache.t4g.micro"
-shared_redis_cluster_node_type = "cache.t4g.small"
+shared_redis_cluster_node_type = "cache.t4g.large"
```

Comment:
There is no `cache.t4g.large` instance type. The burstable cache instances stop at `cache.t4g.medium`. You have to switch up to `cache.m6g.large` if you need anything bigger.